fs/fstest/file_unix.go

```diff
@@ -2,11 +2,29 @@
 
 package fstest
 
-import "github.com/containerd/continuity/sysx"
+import (
+	"path/filepath"
+	"time"
+
+	"github.com/containerd/continuity/sysx"
+	"golang.org/x/sys/unix"
+)
 
 // SetXAttr sets the xatter for the file
 func SetXAttr(name, key, value string) Applier {
 	return applyFn(func(root string) error {
 		return sysx.LSetxattr(name, key, []byte(value), 0)
 	})
 }
+
+// ChtimeNoFollow changes access and mod time of file without following symlink
+func ChtimeNoFollow(name string, t time.Time) Applier {
```

Comment:
`Lchtime`? Also, shouldn't this have args for both?

Reply:
What args? Do you mean merging this into `Chtime` as `func Chtime(name string, t time.Time, followSymlink book) Applier`?

Reply:
This sets both times (atime, mtime) to the same value. Most `Chtimes` function set both.

Reply:
`func Chtime(name string, atime, mtime time.Time, followSymlink bool)` would be fine?

Reply:
Why do you need `followSymlink` here? Check the `os` package conventions: they have a `Xxx` and `Lxxx` for each function. The `os` package already has `Chtimes`.